### PR TITLE
Adding MongoDB loader.

### PR DESCRIPTION
**Pull request by ejouini**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -123,7 +129,18 @@
 			<version>${secure-properties.version}</version>
 			<classifier>mule-plugin</classifier>
 		</dependency>
-	</dependencies>
+		<dependency>
+            <groupId>com.mulesoft.connectors</groupId>
+            <artifactId>mule-mongodb-connector</artifactId>
+            <version>6.3.0</version>
```
Comment (DavoCoder) [inline pom.xml:135]: Please externalize this version in the <properties> section of the pom.xml

File: pom.xml [hunk 2]
```diff
@@ -123,7 +129,18 @@
...
+            <classifier>mule-plugin</classifier>
+        </dependency>
+    	<dependency>
+            <groupId>org.mongodb</groupId>
+            <artifactId>mongodb-driver-legacy</artifactId>
+            <version>4.0.4</version>
```
Comment (DavoCoder) [inline pom.xml:141]: Please externalize this version in the <properties> section of the pom.xml

File: src/main/mule/global.xml [hunk 3]
```diff
@@ -74,4 +75,11 @@ http://www.mulesoft.org/schema/mule/secure-properties http://www.mulesoft.org/sc
 			</http:authentication>
 		</http:request-connection>
 	</http:request-config>
+	<mongo:config name="MongoDB_Config" doc:name="MongoDB Config" doc:id="c1a77712-a55d-4bc3-ba88-70401defab33" >
```
Comment (DavoCoder) [inline src/main/mule/global.xml:78]: Please move this configuration to the specific loader file (loader-mongodb.xml)

File: src/main/resources/properties/app-dev.yaml [hunk 4]
```diff
@@ -2,21 +2,21 @@ http:
   port: "8081"
   
 poller:
-  enabled: "false"
```
Comment (DavoCoder) [inline src/main/resources/properties/app-dev.yaml:5]: Let's keep this as false

File: src/main/resources/properties/app-dev.yaml [hunk 5]
```diff
@@ -2,21 +2,21 @@ http:
...
+  enabled: "true"
   frequency: 
-    cron: "0 0 0 * * ? *"
```
Comment (DavoCoder) [inline src/main/resources/properties/app-dev.yaml:7]: Let's keep this with the original cron

File: src/main/resources/properties/app-dev.yaml [hunk 6]
```diff
@@ -2,21 +2,21 @@ http:
...
-  enabled: "false"
+  enabled: "true"
   frequency: 
-    cron: "0 0 0 * * ? *" 
+    cron: "0 0/1 * 1/1 * ? *" 
     #Sample for dev "0 17 * * * ? *" At second :00 of minute :17 of every hour
     #Sample for prod "0 0 0 * * ? *" At 00:00:00am every day
     #https://www.freeformatter.com/cron-expression-generator-quartz.html
-    timezone: "GMT-3"
+    timezone: "GMT+1"
```
Comment (DavoCoder) [inline src/main/resources/properties/app-dev.yaml:11]: Let's keep the original timezone

File: src/main/resources/properties/app-dev.yaml [hunk 7]
```diff
@@ -2,21 +2,21 @@ http:
...
 auth:
   mode: "platform-credentials" # valid options are: platform-credentials or connected-app-credentials
-
+  
 aggregation:
   raw: "false"
   
 loader:
-  strategy: "logger"
```
Comment (DavoCoder) [inline src/main/resources/properties/app-dev.yaml:19]: Let's keep the default loader as "logger"

File: src/main/resources/properties/app-dev.yaml [hunk 8]
```diff
@@ -44,7 +44,7 @@ loader:
 
 # Default value: "all"
   
-collectors: "all"
```
Comment (DavoCoder) [inline src/main/resources/properties/app-dev.yaml:47]: Let's keep the collectors as "all"

File: src/main/resources/properties/app-dev.yaml [hunk 9]
```diff
@@ -55,9 +55,15 @@ splunk:
...
-    
+
+mongodb:
+  host: "127.0.0.1"
+  port: "27017"
+  database: "matrixdb"
+  collection: "metrics"
+            
 elk:
-  host: "localhost"
```
Comment (DavoCoder) [inline src/main/resources/properties/app-dev.yaml:60]: Let's keep this as localhost

File: src/main/resources/properties/secure/app-dev.yaml [hunk 10]
```diff
@@ -1,10 +1,10 @@
 auth:
   
-  orgId: "![yqKEWZ57M3XUasumfcwhVykLsfUM6yOEv2oEJ04vxZc=]"
```
Comment (DavoCoder) [inline src/main/resources/properties/secure/app-dev.yaml:3]: Let's be careful with these values, seem to be related to a real org, we included fake orgId, username/pass, let's keep the original ones

File: src/main/resources/properties/secure/app-dev.yaml [hunk 11]
```diff
@@ -1,10 +1,10 @@
...
+  orgId: "![MZrkNkhH/Z9wpUqTaXJ/tLmHXPLMtMTAs6/5CfaR4A4bVDlv1ye1FkyxVvHGChry]"
   
   # If auth mode = platform-credentials (ignored, otherwise):
-  username: "![Zjv8nFIUwDlayZ6O7DP31p1zQRfrwfjbkmi5DyNiD38=]"
```
Comment (DavoCoder) [inline src/main/resources/properties/secure/app-dev.yaml:6]: Let's be careful with these values, seem to be related to a real org, we included fake orgId, username/pass, let's keep the original ones

File: src/main/resources/properties/secure/app-dev.yaml [hunk 12]
```diff
@@ -17,8 +17,12 @@ splunk:
   token: "![yqKEWZ57M3XUasumfcwhV5A3JMblY6G4MCwHM5j5VuA=]"
   
 elk:
-  user: "user"
-  password: "password"
+  user: "![cClkzNzePN9/3L4Ag8XzsQ==]"
```
Comment (DavoCoder) [inline src/main/resources/properties/secure/app-dev.yaml:20]: Let's be careful with these values, seem to be related to a real ELK instance, let's keep the original ones

**Review by DavoCoder — CHANGES_REQUESTED** [review]
Please review my inline comments

File: pom.xml (hunk 1 above)
Comment (ejouini) [reply pom.xml]: Done.

File: pom.xml (hunk 2 above)
Comment (ejouini) [reply pom.xml]: Done.

**Comment by panizzag** [comment]
@damiancalabresi This PR contains pending changes to review. Could you please take a look? Thanks

**Pull request merged**